src/test/java/com/opentok/test/OpenTokTest.java

```diff
@@ -681,29 +676,249 @@ public void testListArchives() throws OpenTokException {
+                                "            \"size\" : 78499758,\n" +
+                                "            \"status\" : \"available\",\n" +
+                                "            \"url\" : \"http://tokbox.com.archive2.s3.amazonaws.com/123456%2F30b3ebf" +
+                                "1-ba36-4f5b-8def-6f70d9986fe9%2Farchive.mp4?Expires=1395188695&AWSAccessKeyId=AKIAI6" +
+                                "LQCPIXYVWCQV6Q&Signature=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx\"\n" +
+                                "          }, {\n" +
+                                "            \"createdAt\" : 1394396753000,\n" +
+                                "            \"duration\" : 24,\n" +
+                                "            \"id\" : \"b8f64de1-e218-4091-9544-4cbf369fc238\",\n" +
+                                "            \"name\" : \"showtime again\",\n" +
+                                "            \"partnerId\" : 123456,\n" +
+                                "            \"reason\" : \"\",\n" +
+                                "            \"sessionId\" : \"SESSIONID\",\n" +
+                                "            \"size\" : 2227849,\n" +
+                                "            \"status\" : \"available\",\n" +
+                                "            \"url\" : \"http://tokbox.com.archive2.s3.amazonaws.com/123456%2Fb8f64de" +
+                                "1-e218-4091-9544-4cbf369fc238%2Farchive.mp4?Expires=1395188695&AWSAccessKeyId=AKIAI6" +
+                                "LQCPIXYVWCQV6Q&Signature=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx\"\n" +
+                                "          }, {\n" +
+                                "            \"createdAt\" : 1394321113000,\n" +
+                                "            \"duration\" : 1294,\n" +
+                                "            \"id\" : \"832641bf-5dbf-41a1-ad94-fea213e59a92\",\n" +
+                                "            \"name\" : \"showtime\",\n" +
+                                "            \"partnerId\" : 123456,\n" +
+                                "            \"reason\" : \"\",\n" +
+                                "            \"sessionId\" : \"SESSIONID\",\n" +
+                                "            \"size\" : 42165242,\n" +
+                                "            \"status\" : \"available\",\n" +
+                                "            \"url\" : \"http://tokbox.com.archive2.s3.amazonaws.com/123456%2F832641b" +
+                                "f-5dbf-41a1-ad94-fea213e59a92%2Farchive.mp4?Expires=1395188695&AWSAccessKeyId=AKIAI6" +
+                                "LQCPIXYVWCQV6Q&Signature=xxxxxxxxxxxxxxxxxxxxxxxxxxxxxx\"\n" +
+                                "          } ]\n" +
+                                "        }")));
+        ArchiveList archives = sdk.listArchives(sessionId);
+        assertNotNull(archives);
+        assertEquals(6, archives.size());
```

Comment:
Could you please explain this?

Reply:
refer https://github.com/opentok/Opentok-Java-SDK/pull/150#discussion_r212039906